charts/opentelemetry-operator/templates/clusterrole.yaml

```diff
@@ -86,6 +87,15 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+      - namespaces
```

Comment:
@zarbis I see where `nodes` is being added [here](https://github.com/open-telemetry/opentelemetry-operator/blob/c65629b3b9fcb722e770eb58b73ea4f5cc8bc63f/internal/manifests/collector/parser/processor/processor_k8sattributes.go#L78-L102), but I don't see `namespaces`.

Reply:
@TylerHelmuth a bit [above](https://github.com/open-telemetry/opentelemetry-operator/blob/c65629b3b9fcb722e770eb58b73ea4f5cc8bc63f/internal/manifests/collector/parser/processor/processor_k8sattributes.go#L56-L60).